specification/deviceregistry/DeviceRegistry.Management/assets.tsp

```diff
@@ -75,7 +75,7 @@ model AssetProperties {
   manufacturer?: string;
 
   @doc("Asset manufacturer URI.")
-  manufacturerUri?: string;
+  manufacturerUri?: url;
```

Comment:
Please revert, this is a breaking change. Type change requires a new API version.
The same change broke our APIs in the past: https://github.com/Azure/azure-rest-api-specs/pull/29319

Reply:
fixed